src/chttpd/src/chttpd_db.erl

```diff
@@ -505,148 +570,179 @@ db_req(#httpd{method='POST', path_parts=[_, <<"_bulk_get">>],
             #doc_query_args{
                 options = Options
             } = bulk_get_parse_doc_query(Req),
-            AcceptJson =  MochiReq:accepts_content_type("application/json"),
+            AcceptJson = MochiReq:accepts_content_type("application/json"),
             AcceptMixedMp = MochiReq:accepts_content_type("multipart/mixed"),
             AcceptRelatedMp = MochiReq:accepts_content_type("multipart/related"),
             AcceptMp = not AcceptJson andalso (AcceptMixedMp orelse AcceptRelatedMp),
             case AcceptMp of
                 false ->
                     {ok, Resp} = start_json_response(Req, 200),
                     send_chunk(Resp, <<"{\"results\": [">>),
-                    lists:foldl(fun(Doc, Sep) ->
-                        {DocId, Results, Options1} = bulk_get_open_doc_revs(Db, Doc,
-                            Options),
-                        bulk_get_send_docs_json(Resp, DocId, Results, Options1, Sep),
-                        <<",">>
-                    end, <<"">>, Docs),
+                    lists:foldl(
+                        fun(Doc, Sep) ->
+                            {DocId, Results, Options1} = bulk_get_open_doc_revs(
+                                Db,
+                                Doc,
+                                Options
+                            ),
+                            bulk_get_send_docs_json(Resp, DocId, Results, Options1, Sep),
+                            <<",">>
+                        end,
+                        <<"">>,
+                        Docs
+                    ),
                     send_chunk(Resp, <<"]}">>),
                     end_json_response(Resp);
                 true ->
                     OuterBoundary = bulk_get_multipart_boundary(),
-                    MpType = case AcceptMixedMp of
-                        true ->
-                            "multipart/mixed";
-                        _ ->
-                            "multipart/related"
-                    end,
-                    CType = {"Content-Type", MpType ++ "; boundary=\"" ++
-                                          ?b2l(OuterBoundary) ++  "\""},
+                    MpType =
+                        case AcceptMixedMp of
+                            true ->
+                                "multipart/mixed";
+                            _ ->
+                                "multipart/related"
+                        end,
+                    CType =
+                        {"Content-Type",
```

Comment:
-1. As human I would split it on separate lines:

```
{
     "Content-Type",
      MpType ++ "; boundary=\"" ++ ?b2l(OuterBoundary) ++ "\""
}       
```

OR
```
{
     "Content-Type",
      MpType 
          ++ "; boundary=\"" 
          ++ ?b2l(OuterBoundary) 
          ++ "\""
}
```

Reply:
Done in 81d5295